Core/include/Acts/Surfaces/detail/AlignmentHelper.hpp

```diff
@@ -0,0 +1,81 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <utility>
+#include <vector>
+#include "Acts/Utilities/Definitions.hpp"
+
+namespace Acts {
+
+namespace detail {
+
+/// @brief Helper methods for calculating derivatives related with surface
+/// alignment
+namespace AlignmentHelper {
```

Comment:
I am not a big fan of these additional nested namespaces. By being in the `Acts::detail` namespace it is already clear that these are helper functions and types. The names are sufficiently clear to not need additional specification.

Reply:
I agree, thus removed. :smile: